src/ogma/features/styles.ts

```diff
@@ -382,7 +382,7 @@ export class StylesViz {
         opacity: FILTER_OPACITY,
         layer: (node): number => {
           // if the node is part of a virtual node, it should be on top
-          if (node.getMetaNode() !== undefined) {
+          if (node.getMetaNode() !== null) {
```

Comment:
@davidrapin for me this is the fix of a regression that caused the bug since node grouping was implemented. We could only fix this and it would work.